rfc/wip-tform/index.md

```diff
@@ -0,0 +1,679 @@
+system with the name of the "input" value gives the input dimension.  If the value of "input" is an array, it's
+length gives the input imension, otherwise it is given by the length of "axes" for the coordinate system with
+the name of the "input".  If the value of "output" is an array, it's length gives the output dimension,
+otherwise it is given by the length of "axes" for the coordinate system with the name of the "output".
+
+#### <a name="identity">identity</a>
+
+`identity` transformations map input coordinates to output coordinates without modification. The position of
+the ith axis of the output coordinate system is set to the position of the ith axis of the input coordinate
+system. `identity` transformations are invertible.
+
+
+#### <a name="mapAxis">mapAxis</a>
+
+`mapAxis` transformations describe axis permutations as a mapping of axis names. Transformations MUST include a `mapAxis` field
+whose value is an object, all of whose values are strings. If the object contains `"x":"i"`, then the transform sets the value 
+of the output coordinate for axis "x" to the value of the coordinate of input axis "i" (think `x = i`). For every axis in its output coordinate
+system, the `mapAxis` MUST have a corresponding field. For every value of the object there MUST be an axis of the input
+coordinate system with that name. Note that the order of the keys could be reversed.
+
+
+#### <a name="translation">translation</a>
+
+`translation` transformations are special cases of affine transformations. When possible, a 
+translation transformation should be preferred to its equivalent affine. Input and output dimensionality MUST be
+identical and MUST equal the the length of the "translation" array (N). `translation` transformations are
+invertible.
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The path to a zarr-array containing the translation parameters.
+	The array at this path MUST be 1D, and its length MUST be `N`.</dd>
+  <dt><strong>scale</strong></dt>
+  <dd> 	The scale parameters stored as a JSON list of numbers. The list MUST have length `N`.</dd>
+</dl>
+
+
+#### scale
+
+`scale` transformations are special cases of affine transformations. When possible, a scale transformation
+SHOULD be preferred to its equivalent affine. Input and output dimensionality MUST be identical and MUST equal
+the the length of the "scale" array (N). Values in the `scale` array SHOULD be non-zero; in that case, `scale`
+transformations are invertible.
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The path to a zarr-array containing the scale parameters.
+	The array at this path MUST be 1D, and its length MUST be `N`.</dd>
+  <dt><strong>scale</strong></dt>
+  <dd> 	The scale parameters stored as a JSON list of numbers. The list MUST have length `N`.</dd>
+</dl>
+
+
+#### <a name="affine">affine</a>
+
+`affine`s are [matrix transformations](#matrix-transformations) from N-dimensional inputs to M-dimensional outputs are represented at `(N)x(M+1)`
```

Comment:
I think there is an error: if N are the input and M are the output, then the matrix should have M rows and N + 1 columns, hence it should be (M)x(N+1).

Reply:
If (N)x(M+1) is changed to (M)x(N+1) then it needs to be changed also in the text below.